### ISSUE TITLE
Error building Spine-UE4 3.7-beta-cpp on Mac

### ISSUE THREAD

**Issue by antidamage** [EsotericSoftware/spine-runtimes#1150]
When I attempt to build the latest Spine plugin for UE4 on Mac I get the following errors:

```
/Users/justincook/Dropbox/Work/Games/Projects/PoppyAndBuddy419/Plugins/SpinePlugin/Source/SpinePlugin/Private/SpineSkeletonAnimationComponent.cpp:42:38: error: reference to 'EventType' is ambiguous

/Users/justincook/Dropbox/Work/Games/Projects/PoppyAndBuddy419/Plugins/SpinePlugin/Source/SpinePlugin/Public/spine-cpp/src/spine/AnimationState.cpp:49:55: error: reference to 'EventType' is ambiguous

/Users/justincook/Dropbox/Work/Games/Projects/PoppyAndBuddy419/Plugins/SpinePlugin/Source/SpinePlugin/Public/spine-cpp/src/spine/SkeletonBounds.cpp:202:1: error: reference to 'Polygon' is ambiguous
```

I'm uncertain what to do as I'm flying pretty blind when it comes to code stuff, especially on Mac. I had no problems building it for Windows though.

Any advice welcome.

**Comment by antidamage** [comment]
I was able to fix it by adding spine:: in front of the three types mentioned, forcing the type to be unambiguous.

**Comment by badlogic** [comment]
Which UE4 version are you using?

**Comment by badlogic** [comment]
Thanks for reporting, this is fixed in the 3.7-beta and 3.7-beta-cpp branches now.
